client/src/components/Details/Timer.js

```diff
@@ -0,0 +1,17 @@
+import React, { useState, useEffect } from "react";
+
+export const Timer = ({ persist_content, is_open }) => {
```

Comment:
If this only exists for the sake of the story, please declare it inside the story (and don't export it). Don't want someone using it for something outside the story later on. It'd unintentionally couple the story and wherever else it gets used. We want to limit coupling between a given story and src, as much as possible, to just the specific component the story demos.

Reply:
Done!